src/Encamina.Enmarcha.SemanticKernel.Connectors.Document/IEnmarchaDocumentConnector.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.SemanticKernel.Plugins.Document;
+
+namespace Encamina.Enmarcha.SemanticKernel.Connectors.Document;
+
+/// <inheritdoc/>
+public interface IEnmarchaDocumentConnector : IDocumentConnector
```

Comment:
Could we consider whether including "Enmarcha" in the interface name is necessary? For consistency, it doesn’t seem to be used elsewhere in the code, aside from namespaces. I can’t think of a perfect alternative right now, but perhaps something like `IFileDocumentConnector` or `IDocumentHandler` could work, but I’m not very inspired at the moment :(

Reply:
The name `IEnmarchaDocumentConnector` indicates that it is our extended version of the semantic kernel `IDocumentConnector`... if we change the name too much it would not be so clear :(

Reply:
Maybe `IDocumentConnectorExtended`? Though I’m not thrilled about the name. I’ll mark it as resolved :)